source/frontend/src/features/contacts/contact/utils/contactUtils.ts

```diff
@@ -23,9 +23,14 @@ export const onValidateOrganization = (
   const errors = {} as any;
   try {
     // combine yup schema validation with custom rules
-    if (!hasEmail(values) && !hasPhoneNumber(values) && !hasAddress(values)) {
+    if (
+      !hasEmail(values) &&
+      !hasPhoneNumber(values) &&
+      !hasAddress(values) &&
+      (values.isDisabled === 'false' || values.isDisabled === false)
+    ) {
```

Comment:
if you follow the pattern I described in my other comment, then here you would only need to check for the string "false"